web/server/README.md

```diff
@@ -112,14 +112,14 @@ Example:
 
 For information how to configure the slaves to use TLS, check [securing the communication](../../streaming#securing-streaming-communications) in the streaming documentation. There you will find additional details on the expected behavior for client and server nodes, when their respective TLS options are enabled.
 
-When we define the use of SSL in a Netdata master, we are defining the behavior of the whole software, and case there is a conflict between the Netdata ports, the server will always assume the last rule given, for example, the next rules
+When we define the use of SSL in a Netdata agent for different ports, the Netdata will apply the TLS rule for the specific port that is assigned, for example, in the next configuration line
```

Comment:
When we define the use of SSL in a Netdata agent for different ports, Netdata will apply the behavior specified on each port. For example, using the configuration line below:

Reply:
I changed the documentation as you asked, and I will bring them in the next commit.